src/treetensornetworks/ttn.jl

```diff
@@ -45,8 +45,8 @@ function TTN(g::AbstractGraph, args...; kwargs...)
   return TTN(Float64, g, args...; kwargs...)
 end
 
-function TTN(eltype::Type{<:Number}, graph::AbstractGraph, args...; kwargs...)
-  itensor_network = ITensorNetwork(eltype, graph; kwargs...)
+function TTN(eltype::Type{<:Number}, graph::AbstractGraph, args...; link_space=1, kwargs...)
+  itensor_network = ITensorNetwork(eltype, graph; link_space, kwargs...)
```

Comment:
Sorry to be a bother, but probably we should define `link_space=1` as the default at the `ITensorNetwork` level, if that doesn't cause issues. It would be pretty arbitrary/inconsistent if `TTN` had that behavior but `ITensorNetwork`didn't.

Reply:
Sure, no problem. I was thinking the same, but again wasn't sure whether this may break other functionality.

Reply:
Moving it to `ITensorNetworks` is breaking at least the `mincut` tests in `test_itensornetworks.jl`.

Reply:
Ok, I would be biased towards investigating failing tests, and either modifying them or disabling them, since setting the default of the `ITensorNetwork` constructor to `link_space=1` is probably the right way to go.

Reply:
It looks like it's not passing the kwargs properly for some signatures, I'll take care of it.